idmapping-rest/src/main/java/org/uniprot/api/idmapping/output/converter/uniprotkb/UniProtKBEntryPairFastaMessageConverter.java

```diff
@@ -1,13 +1,22 @@
 package org.uniprot.api.idmapping.output.converter.uniprotkb;
 
+import java.io.IOException;
+import java.io.OutputStream;
+
 import org.uniprot.api.common.concurrency.Gatekeeper;
 import org.uniprot.api.idmapping.model.UniProtKBEntryPair;
 import org.uniprot.api.idmapping.output.converter.AbstractEntryPairFastaConverter;
+import org.uniprot.api.rest.output.context.MessageConverterContext;
+import org.uniprot.core.fasta.UniProtKBFasta;
+import org.uniprot.core.fasta.impl.UniProtKBFastaBuilder;
 import org.uniprot.core.parser.fasta.uniprot.UniProtKBFastaParser;
 import org.uniprot.core.uniprotkb.UniProtKBEntry;
 
 public class UniProtKBEntryPairFastaMessageConverter
         extends AbstractEntryPairFastaConverter<UniProtKBEntryPair, UniProtKBEntry> {
+
+    private static final ThreadLocal<Boolean> TL_SUBSEQUENCE = new ThreadLocal<>();
```

Comment:
We don't need to use ThreadLocal here
i have seen usage, we are creating new instance and instance it self is not shared by multi thread